pkg/admissioncontroller/webhook/admission/authenticationconfig/handler.go

```diff
@@ -191,10 +170,7 @@ func (h *Handler) admitConfigMap(ctx context.Context, request admission.Request)
 			if err := gardencorev1beta1.Convert_v1beta1_KubeAPIServerConfig_To_core_KubeAPIServerConfig(shoot.Spec.Kubernetes.KubeAPIServer, coreKubeAPIServerConfig, nil); err != nil {
 				return admission.Errored(http.StatusUnprocessableEntity, err)
 			}
-			disallowedIssuers, err = getDisallowedIssuers(coreKubeAPIServerConfig)
-			if err != nil {
-				return admission.Errored(http.StatusUnprocessableEntity, fmt.Errorf("error getting disallowed issuers: %w", err))
-			}
+			disallowedIssuers = getDisallowedIssuers(coreKubeAPIServerConfig)
```

Comment:
Does not checking an error lead to a change in behavior?

Reply:
In [8ef2b688](https://github.com/gardener/gardener/pull/10647/commits/8ef2b688fbdb27b6fe341e279314ac8386d23a6e) the `getDisallowedIssuers` function was changed to no longer return an error.
Before that it returned an error when there were no disallowed issuers, which was a bug as well since it is viable for the disallowed issuers slice to be empty.

Reply:
I have updated the description now to include this information